aldryn_config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from aldryn_client import forms
+
+
+class Form(forms.BaseForm):
+    editor_theme = forms.CharField(
+        'Custom editor theme, (e.g. "github", default: "twilight")',
+        required=False,
+    )
+    editor_mode = forms.CharField(
+        'Custom editor mode (e.g. "javascript", default: "html")',
+        required=False,
+    )
+    enable_search = forms.BooleanField(
```

Comment:
I thought about adding this but my concern is that this presents a security issue and should probably only be enabled by developers. That said, I'm ok to leave it but just add a warning.